### PR TITLE
add api documentation for request/revoke token endpoints

### PR DESCRIPTION
**Pull request by cpapazian**
@yozlet i wrote moar docs!

### CODE REVIEW

**Comment by cpapazian** [comment]
https://github.com/18F/myusa/issues/441

File: public/api.md [hunk 1]
```diff
@@ -177,7 +177,7 @@ If your app uses the MyUSA API, you'll need to keep track of new versions to ens
 
 To sign in to your app, users should be shown a "Connect with MyUSA" button. (See the [Branding section](#branding) for details and code.) The button should link to your app's custom sign in URL, which you can construct from this format:
 ```
-    https://my.usa.gov/users/sign_in?login_reqired=true&client_id=ABCD
```
Comment (yozlet) [inline public/api.md]: I should have run the thing. I did not run the thing. Next time, I must remember to run the thing.

**Comment by yozlet** [comment]
@cpapazian OMG you totally did :+1: :hamburger: :shipit:

**Comment by jackiekazil** [comment]
Is there a reason why branches are not deleted after they are merged? (it seems like we have a lot of branches and I can't imagine there are that many streams of work going on all at once.) :-)

**Pull request merged**